File: Copy/app/src/main/java/com/deadely/copy/MainActivity.java
package com.deadely.copy;

import android.app.Activity;
import android.os.Bundle;
import android.widget.GridView;

public class MainActivity extends Activity {

    GridView gridview;

    public static String[] mvNameList = {
            "Урфин Джюс и его деревянные солдаты",
            "Квартира",
            "Остров собак",
            "Лига справедливости",
            "Мэри и Макс",
            "Песнь моря",
            "Призрачная нить",
            "Шарада",
            "Кошмар перед Рождеством",
    };
    public static int[] mvImages = {
            R.drawable.p1,
            R.drawable.p2,
            R.drawable.p3,
            R.drawable.pi4,
            R.drawable.p5,
            R.drawable.p6,
            R.drawable.p7,
            R.drawable.p8,
            R.drawable.p9,};

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        gridview = (GridView) findViewById(R.id.grid_view);

        gridview.setAdapter(new DataAdapter(this, mvNameList, mvImages));
    }


}

File: itlen/app/src/main/java/com/deadely/itl_en/main/main/view/MainActivity.kt
package com.deadely.itl_en.main.main.view

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.viewpager.widget.ViewPager
import com.deadely.itl_en.R
import com.deadely.itl_en.main.main.ViewPagerAdapter
import kotlinx.android.synthetic.main.activity_main.*


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        initView()
    }

    private fun initView() {
        vp_main.adapter = ViewPagerAdapter(supportFragmentManager)
        vp_main.addOnPageChangeListener(object : ViewPager.OnPageChangeListener {
            override fun onPageScrollStateChanged(state: Int) {

            }

            override fun onPageScrolled(position: Int, positionOffset: Float, positionOffsetPixels: Int) {

            }

            override fun onPageSelected(position: Int) {
                when (position) {
                    0 -> bottom_navigation.menu.findItem(R.id.mnu_stats).isChecked = true
                    1 -> bottom_navigation.menu.findItem(R.id.mnu_study).isChecked = true
                    2 -> bottom_navigation.menu.findItem(R.id.mnu_vocab).isChecked = true
                }
            }
        })

//        val navController = Navigation.findNavController(this, R.id.nav_frag)
        bottom_navigation.setOnNavigationItemSelectedListener { menuItem ->
            if (!menuItem.isChecked) {
//                navController.navigate(menuItem.itemId)
                when (menuItem.itemId) {
                    R.id.mnu_stats -> vp_main.currentItem = 0
                    R.id.mnu_study -> vp_main.currentItem = 1
                    R.id.mnu_vocab -> vp_main.currentItem = 2
                }
            }
            true
        }
    }
}